### PR TITLE
Create zh_tw.json

### PR DESCRIPTION
**Pull request by dirtTW**
Add Traditional Chinese localization

### CODE REVIEW

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  "text.autoconfig.worldtools.option.general.compressLevel": "將擷取存檔為 zip 檔案",
+  "text.autoconfig.worldtools.option.general.compressLevel.@Tooltip": "當擷取完成後，它將會被存檔為 zip 檔案。",
+  "text.autoconfig.worldtools.option.render": "繪製",
+  "text.autoconfig.worldtools.option.render.accentColor": "強調色 (RGB 十六進位)",
+  "text.autoconfig.worldtools.option.render.captureBarColor": "擷取資訊列顏色 (RGB 十六進位)",
+  "text.autoconfig.worldtools.option.render.captureBarStyle": "擷取資訊列樣式",
+  "text.autoconfig.worldtools.option.render.unscannedContainerColor": "遺漏容器顏色 (RGB 十六進位)",
+  "text.autoconfig.worldtools.option.render.fromCacheLoadedContainerColor": "從快取載入的容器顏色 (RGB 十六進位)",
+  "text.autoconfig.worldtools.option.render.unscannedEntityColor": "遺漏實體顏色 (RGB 十六進位)",
+  "text.autoconfig.worldtools.option.render.fromCacheLoadedEntityColor": "從快取載入的實體顏色 (RGB 十六進位)",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "text.autoconfig.worldtools.option.render.accentColor": "強調色（RGB 十六進位）",
  "text.autoconfig.worldtools.option.render.captureBarColor": "擷取資訊列顏色（RGB 十六進位）",
  "text.autoconfig.worldtools.option.render.captureBarStyle": "擷取資訊列樣式",
  "text.autoconfig.worldtools.option.render.unscannedContainerColor": "遺漏容器顏色（RGB 十六進位）",
  "text.autoconfig.worldtools.option.render.fromCacheLoadedContainerColor": "從快取載入的容器顏色 (RGB 十六進位）",
  "text.autoconfig.worldtools.option.render.unscannedEntityColor": "遺漏實體顏色（RGB 十六進位）",
  "text.autoconfig.worldtools.option.render.fromCacheLoadedEntityColor": "從快取載入的實體顏色（RGB 十六進位）",
```

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  "text.autoconfig.worldtools.option.render.progressBarColor": "進度條顏色",
+  "text.autoconfig.worldtools.option.render.progressBarStyle": "進度條樣式",
+  "text.autoconfig.worldtools.option.render.progressBarTimeout": "進度條逾時 (毫秒)",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "text.autoconfig.worldtools.option.render.progressBarTimeout": "進度條逾時（毫秒）",
```

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  "worldtools.capture.saved.levelData": "已儲存 %s 的 %s",
+  "worldtools.capture.saved.mapData": "已儲存維度 %s 中的地圖",
+  "worldtools.capture.saved.metadata": "已儲存 %s 的中繼資料",
+  "worldtools.capture.saved.player": "已儲存維度 %s 中 %s 的玩家 %s",
+  "worldtools.capture.saved.player.anonymized": "已儲存維度 %s 中的玩家 %s",
+  "worldtools.capture.saved.statistics": "已儲存 %s 的統計資料",
+  "worldtools.capture.to_directory": " 到儲存資料夾 ",
+  "worldtools.capture.took": "（耗時 %s）",
+  "worldtools.gui.browser.title": "WorldTools 瀏覽器",
+  "worldtools.gui.capture.existing_world_confirm.message": "已存在名為 \"%s\" 的儲存檔案。要覆蓋它嗎？（將會與現有資料合併）",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "worldtools.gui.capture.existing_world_confirm.message": "已存在名為「%s」的儲存檔案。要覆蓋它嗎？（將與現有資料合併）",
```

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  "worldtools.gui.manager.button.start_download": "開始下載",
+  "worldtools.gui.manager.button.stop_download": "停止下載",
+  "worldtools.gui.manager.title": "WorldTools 管理器",
+  "worldtools.gui.manager.world_name_placeholder": "輸入世界名稱...（預設：%s）",
+  "worldtools.key.categories": "WorldTools",
+  "worldtools.key.open_config": "開啟設定",
+  "worldtools.key.toggle_capture": "快速下載",
+  "worldtools.log.error.already_capturing": "%s 的擷取已在執行！",
+  "worldtools.log.error.failed_to_create_session": "無法為 \"%s\" 建立工作階段！（錯誤：%s）",
+  "worldtools.log.error.failed_to_save_level": "無法儲存 \"%s\" 的 level.dat！（錯誤：%s）",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "worldtools.log.error.failed_to_create_session": "無法為「%s」建立工作階段！（錯誤：%s）",
  "worldtools.log.error.failed_to_save_level": "無法儲存「%s」的 level.dat！（錯誤：%s）",
```

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  "worldtools.log.error.failed_to_visit_file": "無法存取檔案 %s 以建立 zip 檔案！（錯誤：%s）",
+  "worldtools.log.error.failed_to_zip": "無法建立工作階段以壓縮 %s 的擷取！（錯誤：%s）",
+  "worldtools.log.error.not_capturing": "尚未執行任何擷取！",
+  "worldtools.log.error.world_name_too_long": "世界名稱 \"%s\" 太長！（最多 %d 個字元）",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "worldtools.log.error.world_name_too_long": "世界名稱「%s」太長！（最多 %d 個字元）",
```

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+  "worldtools.log.info.singleplayer_capture": "實驗性：在單人遊戲中擷取是實驗性的，可能無法按預期工作！",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "worldtools.log.info.singleplayer_capture": "實驗性：在單人遊戲中擷取是實驗性的，可能無法按預期運作！",
```

File: common/src/main/resources/assets/worldtools/lang/zh_tw.json [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+  "text.autoconfig.worldtools.option.world.worldGenerator.bonusChest": "額外獎勵箱",
+  "text.autoconfig.worldtools.option.world.worldGenerator.generateFeatures": "生成結構",
+  "text.autoconfig.worldtools.option.world.worldGenerator.seed": "種子碼",
+  "text.autoconfig.worldtools.option.world.worldGenerator.type": "世界生成類型",
+  "text.autoconfig.worldtools.option.world.worldGenerator.type.@Tooltip": "VOID：將會生成一個虛空世界。FLAT：將會生成一個平坦的世界。DEFAULT：將會生成一個普通的世界。",
+  "text.autoconfig.worldtools.title": "WorldTools 設定",
+  "worldtools.capture.and": " 和 ",
+  "worldtools.capture.chunk": "%s 個區塊",
+  "worldtools.capture.chunks": "%s 個區塊",
+  "worldtools.capture.click_to_open": "%s（點擊以開啟）",
```
Comment (notlin4) [inline common/src/main/resources/assets/worldtools/lang/zh_tw.json]:
```suggestion
  "worldtools.capture.click_to_open": "%s（按這裡開啟）",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by Avanatiker** [comment]
谢谢 :3

**Pull request merged**